### ISSUE TITLE
Viruses in the downloads from your site

### ISSUE THREAD

**Issue by CADbloke** [reportunit/reportunit#53]
Both the files I downloaded from http://relevantcodes.com/reportunit/ (v1.2 & v1.5 beta) were blocked by my virus scanner - v1.5 for a `Trojan:Win32/Spursint.A`

Perhaps your Wordpress site has been hacked?

**Comment by anshooarora** [comment]
I tested with my personal scanner and also corporate.  I didn't receive any warnings..  There has been another user who has reported this issue but it was eventually fixed for them.  Can you check once again?  I can rebuild the exe and reupload.

**Comment by CADbloke** [comment]
Yes, I can test it. Send me the link when you upload it and j will test it
for you.

Cheers
Ewen


​​
- - - - - - - - -
Ewen Wallace
CAD Bloke
​​
Ewen@CADbloke.com
​​
+61 410 598 070 <+61%20410%20598%20070>
http://www.tvCAD.tv
<https://mail.google.com/mail/u/1/%E2%80%8Bhttp://www.tvCAD.tv>  CAD
drawings, Cable schedules: *SORTED*.
​http://www.CADfindREPLACE.com <http://www.cadfindreplace.com/>
http://CADbloke.com/about/ <http://cadbloke.com/about/>
http://www.linkedin.com/in/cadbloke
Skype: CADbloke
Twitter: @CADbloke <https://twitter.com/cadbloke>
​​
​
<https://mail.google.com/mail/u/1/%E2%80%8Bhttp://www.tvCAD.tv>

​​
Before printing this email remember it's the 21st century - this is not a
typewriter.
This email was sent using 100% recycled electrons.

On 17 Mar 2016, at 15:18, Anshoo Arora <notifications@github.com> wrote:

I tested with my personal scanner and also corporate. I didn't receive any
warnings.. There has been another user who has reported this issue but it
was eventually fixed for them. Can you check once again? I can rebuild the
exe and reupload.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/reportunit/reportunit/issues/53#issuecomment-197686380>

**Comment by CADbloke** [comment]
Win10 Defender is still complaining ... 
![reportunittrojan](https://cloud.githubusercontent.com/assets/147329/13897993/7c773462-ee18-11e5-8e64-feaa1cc14e0a.png)

Link provided by Defender - http://go.microsoft.com/fwlink/?linkid=142185&name=Trojan:Win32/Spursint.A&threatid=2147705703&enterprise=0

**Comment by lboening** [comment]
I ran a scan on my windows 10 pc with Windows Defender, at it is "clean"
![2016-03-19_12-11-32](https://cloud.githubusercontent.com/assets/445481/13900076/e34e4330-edcb-11e5-968d-f93559189b16.png)

**Comment by CADbloke** [comment]
Mine was blocked by Chrome + Windows Defender when it downloaded.

Ditto with Internet Exploder on Windows 7 but typically I couldn't get an useful information from that one.

![reportunitsmartscreen](https://cloud.githubusercontent.com/assets/147329/13935917/231a0a1e-f00e-11e5-824a-752588441223.png)
